.test-infra/jenkins/job_PerformanceTests_SQLIO_Java.groovy

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import CommonJobProperties as common
+
+def now = new Date().format("MMddHHmmss", TimeZone.getTimeZone('UTC'))
```

Comment:
What uses this line?

Reply:
Copied it over from `job_PerformanceTests_BigQueryIO_Java.groovy`, where it is used to create a unique output (write) table for every run.
Since we always write to the same table, `now` variable can be dropped.